### PR TITLE
add some step

### PR DESCRIPTION
**Pull request by chenhonggc**

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ the Pod:
 
     ```shell
     root@redis:/data/redis# apt-get update
-    root@redis:/data/redis# apt-get install procps
+    root@redis:/data/redis# apt-get install -y procps
```
Comment (mistyhacks) [inline content/en/docs/tasks/configure-pod-container/configure-volume-storage.md]: Please remove the `-y`. Admins can add that if they want, but we don't want to guide them to force things to install without examining the dependencies that will also be installed.

/hold

Comment (chenhonggc) [reply content/en/docs/tasks/configure-pod-container/configure-volume-storage.md]: @mistyhacks  Thanks.

**Comment by mistyhacks** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**